File: Ruby/1038.rb
menu = { :"1" => 4.00, :"2" => 4.50, :"3" => 5.00, :"4" => 2.00, :"5" => 1.50}
item, qtd = gets.chomp.split()
puts ("Total: R$ %0.2f" % (menu[item.to_sym] * qtd.to_i))

File: Lua/1038.lua
menu = { ["1"] = 4.00, ["2"] = 4.50, ["3"] = 5.00, ["4"] = 2.00, ["5"] = 1.50}
item = io.read("*n")
qtd = io.read("*n")
print(string.format("Total: R$ %.2f", menu[tostring(item)] * qtd))